giles-eco-requests/src/main/java/edu/asu/diging/gilesecosystem/requests/IStorageRequest.java

```diff
@@ -17,4 +17,20 @@ public interface IStorageRequest extends IRequest {
     public abstract void setFilename(String filename);
 
     public abstract String getFilename();
+    
+    public abstract void setPageNr(int pageNr);
+
+    public abstract int getPageNr();
```

Comment:
see my question about page nr in https://github.com/diging/giles-eco-tardis/pull/1